dynamiqs/utils/general.py

```diff
@@ -171,6 +172,53 @@ def sinm(x: QArrayLike) -> QArray:
     return -0.5j * (expm(1j * x) - expm(-1j * x))
 
 
+def signm(x: QArrayLike) -> QArray:
+    r"""Returns the operator sign function of a hermitian qarray.
+
+    The operator sign is generally dense, and is different from the element-wise
+    sign of the operator.
+
+    Args:
+        x _(qarray-like of shape (..., n, n))_: Square matrix.
+
+    Returns:
+        _(qarray of shape (..., n, n))_ Operator sign function of `x`.
+
+    Note:
+        This function uses [`jnp.linalg.eigh()`](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.eigh.html)
+        to compute the sign of a matrix $A$:
+        $$
+            \mathrm{sign}(A) = U \mathrm{sign}(D) U^\dagger
+        $$
```

Comment:
Can we put the formula in the main description?

Reply:
It's not really a definition, hence why I put it there, but ok.